DeviceWhoAmI.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$id": "http://https://github.com/harp-tech/protocol/DeviceWhoAmI.json",
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "definitions": {
+        "device": {
+            "properties": {
+                "contributor": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string",
+                    "pattern": "^[a-zA-Z][a-zA-Z0-9_]*$"
+                },
+
+                "organization": {
```

Comment:
NuGet also initially separated between "authors" and "owners" but they have since deprecated the latter and assume that only authors exist:
https://learn.microsoft.com/en-us/nuget/reference/nuspec#owners

We can discuss whether it makes sense to keep this, but I would propose instead to have two new entries:
  - `"copyright"`
  - `"license"`

Although the format of `license` might require some discussion so maybe just `copyright` for now would be enough?